build.sbt

```diff
@@ -276,18 +276,24 @@ def buildScalacOptions(scalaVersion: String) = {
     "-language:experimental.macros",
     "-unchecked",
     "-Xfatal-warnings",
-    "-Xxml:-coalescing",
-    "-Xfuture",
-    "-Ywarn-infer-any",
-    "-Ywarn-inaccessible",
-    // "-Ywarn-nullary-unit", // we cannot use this. It interferes with the Uniform Access Principle.
-    // See https://stackoverflow.com/questions/7600910/difference-between-function-with-parentheses-and-without.
-    "-Ywarn-unused-import"
+    "-Xxml:-coalescing"
   )
 
   val scalaVersionSpecificOptions = CrossVersion.partialVersion(scalaVersion) match {
     case Some((2, 12)) =>
       Seq(
+        "-Ywarn-unused:imports",
+        "-Xfuture",
+        "-Ywarn-infer-any",
+        "-Ywarn-inaccessible"
+        // "-Ywarn-nullary-unit", // we cannot use this. It interferes with the Uniform Access Principle.
+        // See https://stackoverflow.com/questions/7600910/difference-between-function-with-parentheses-and-without.
+      )
+    case Some((2, 13)) =>
+      Seq(
+        "-Xlint:inaccessible",
+        "-Xlint:infer-any",
+        "-Xlint:nullary-unit",
         "-Ywarn-unused:imports"
```

Comment:
`-Ywarn-unused:imports` is in both 2.12 and 2.13, should that be moved to the above `commonOptions`?